src/main/java/org/wildfly/security/auth/client/ElytronXmlParser.java

```diff
@@ -1,6 +1,6 @@
 /*
  * JBoss, Home of Professional Open Source.
- * Copyright 2014 Red Hat, Inc., and individual contributors
+ * Copyright 2017 Red Hat, Inc., and individual contributors
```

Comment:
Really sorry - missed this one before but it is quite important.

FYI we set the copyright year based on the day we add the file but then it needs to be left unmodified.